src/Elcodi/Store/GeoBundle/Controller/AddressController.php

```diff
@@ -175,7 +195,8 @@ public function editAction(
     public function newAction(
         AddressInterface $address,
         FormView $formView,
-        $isValid
+        $isValid,
+        $source
```

Comment:
Well, is defined by the route defaults but If you think that's better to define it twice I'll change it.